Initial Commit of MetaBroker

holtchesley opened the pull request:
MetaBroker delegates the generation and execution of actions to the
sub-brokers. It will need to be modified once the interface for nodes
is solidified, so that the nodes can be properly initialized.

Commit:
Initial Commit of MetaBroker

MetaBroker delegates the generation and execution of actions to the
sub-brokers. It will need to be modified once the interface for nodes
is solidified, so that the nodes can be properly initialized.

WesleyAC commented on broker/meta_broker.py:
```diff
@@ -0,0 +1,31 @@
+                      tuples(just('Power',  self.power_broker.step())))
+
+    def execute_step(self, step):
+        broker,action = step[0]
```
I don't think that this works - IIRC step is a tuple?

Commit:
Wesley is indeed right

This patch corrects a bad assignment in the MetaBroker that would have
caused it to crash.
